resources/carrier/en/32.txt

```diff
@@ -15,11 +15,10 @@
 # 1994 by Belgacom, following the launch of the first mobile network.Proximus is
 # a subsidiary of Belgacom.
 # Mobistar has re-branded as Orange on 9 May 2016.
-# Unleashed has 3 brands Mobile Vikings, JIM Mobile and Stievie. Prefix 456
-# phone numbers are mentioned in contact-us page of JIM mobile carrier. So it is
-# more likely that it is assigned to 'JIM Mobile'
+# Unleashed has 2 mobile brands: Mobile Vikings and JIM Mobile. Both of these
+# brands distribute numbers with prefix 456.
 
-32456|JIM Mobile
+32456|Unleashed
```

Comment:
Unleashed is the company and we prefer the brand names instead. As mentioned in the comment above prefix 456 seems to be assigned to JIM Mobile only https://jimmobile.be/nl/contact/ so 456 should be JIM Mobile rather than Unleashed.

Reply:
As stated in the commit message, Unleashed will very soon (probably this month) start distributing 456-prefixed phone numbers for Mobile Vikings too. A similar mention to that on the contact page can be found on [this FAQ page](https://support.vikingco.com/hc/en-us/articles/360000313477-What-will-happen-after-activation-on-the-new-platform-):

> The short number for voicemail remains the same at 1933. If you want to call your voicemail from abroad, bear in mind that the number now is +32 **456** 19 19 33 instead of +32 486 19 33 33.

Since the same prefix will be used for new onboardings for both brands (and MV is the bigger one, by the way), it is wrong to call it JIM Mobile. It would be fine to list "JIM Mobile/Mobile Vikings" if you think "Unleashed" is inappropriate.

Reply:
Let's do "JIM Mobile/Mobile Vikings" as you said since they are the real underlying telecoms who are responsible for the range. Our preference is to avoid parent companies whenever possible.

Could you provide some external link as proof of this change and add it to the commit message too?